src/application/App.tsx

```
@@ -0,0 +1,21 @@
+import type { NextPage } from 'next'
+import type { AppProps } from 'next/app'
+
+import type { ReactElement, ReactNode } from 'react'
+
+import { WithStore } from '@/application/providers/WithStore'
+
+export type NextPageWithLayout<P = {}, IP = P> = {
```

Biome

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)